viostor/virtio_stor_hw_helper.h

```diff
@@ -45,7 +45,7 @@
 #include <srbhelper.h>
 
 FORCEINLINE ULONG
-SrbGetCdbLenght(_In_ PVOID Srb) {
+SrbGetCdbLength(_In_ PVOID Srb) {
```

Comment:
SrbGetCdbLength is defined in srbhelper.h
SrbGetCdbLenght should be renamed to SrbGetCdbLength32. The same as in vioscsi driver:

https://github.com/virtio-win/kvm-guest-drivers-windows/blob/0196cb4779dd6017b735ce601dfd513a56bfa454/vioscsi/helper.h#L54-L56